vm/parser.ts

```diff
@@ -759,8 +764,17 @@ module J2ME {
       var s = this;
       var methods_count = s.readU2();
       var m = this.methods = new Array(methods_count);
+      var ids = this.methodIDs = new Array(methods_count);
       for (var i = 0; i < methods_count; i++) {
         m[i] = s.offset;
+
+        ids[i] = [
```

Comment:
> As long as we need that info to minimally link the methods, it doesn't actually reduce the effectiveness of lazily loading the classes, because we'll load the info at link time anyway.

Yes, but with this change we''re going to load eagerly all the methods of a class, even if we need only a few.